tests/net/mod.rs

```diff
@@ -513,9 +527,13 @@ where
 
         // Note: Closure is not redundant, won't compile without it.
         #[cfg_attr(feature = "cargo-clippy", allow(redundant_closure))]
-        let mut net = VirtualNet::new(self.node_ids, self.num_faulty as usize, rng, move |node| {
-            cons(node)
-        })?;
+        let mut net = VirtualNet::new(
+            self.node_ids,
+            self.num_faulty as usize,
+            rng,
+            self.observer_id,
+            move |node| cons(node),
```

Comment:
Isn't this equivalent to just `cons`?

Reply:
No, it doesn't compile without the closure.